src/nav2_localization/amcl/src/amcl_node.cpp

```diff
@@ -45,6 +45,25 @@
 
 using namespace amcl;
 
+// TODO (mhpanah): Factor out strutils and put it in util directory
+class strutils
+{
+public:
+  static std::string stripLeadingSlash(const std::string& in);
+};
+
+std::string 
```

Comment:
Return type should be on the same line if it fits.

Reply:
Fixed.